src/main/java/step2/validation/InputViewValidator.java

```diff
@@ -0,0 +1,42 @@
+package step2.validation;
+
+import step2.util.StringUtil;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+public class InputViewValidator {
+
+    private static final int MINIMUM = 1;
+    private static final int MAXIMUM = 45;
+
+    public static void prizeCountValidate(int count) {
+        if (count <= 0) {
+            throw new IllegalArgumentException("금액이 부족합니다.");
+        }
+    }
+
+    public static void winningLottoValidate(String number) {
+        List<Integer> list = StringUtil.stringToList(number);
+        if(list.size() != 6){
+            throw new IllegalArgumentException("로또 번호 갯수가 유효하지 않습니다.");
+        }
+
+
+        Stream<Integer> checkDuplicate = list.stream().distinct();
+        if(checkDuplicate.count() != 6){
+            throw new IllegalArgumentException("중복된 로또 번호가 존재합니다.");
+        }
+
+        for (int i = 0; i < list.size(); i++) {
+            validateNumber(list.get(i));
+        }
```

Comment:
전체적으로 향상된 for문 사용이 없으신것 같네요. 인덱스를 활용한 for문보다 훨씬 빠르고 코드가 간결해집니다. 🙂 
```suggestion
        for (Integer integer : list) {
            validateNumber(integer);
        }
```